app/views/UTRSentByPostView.scala.html

```diff
@@ -21,17 +21,22 @@
 main_template: MainTemplate
 )
 
-@()(implicit request: Request[_], messages: Messages)
+@(isAgent: Boolean)(implicit request: Request[_], messages: Messages)
 
 @main_template(
 title = messages("UTRSentByPost.title")
 ) {
 
+@components.back_link()
+
 @components.heading("UTRSentByPost.heading")
 
 <p>@messages("UTRSentByPost.paragraph1")</p>
 
-<p><a id="help-link" href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/trusts" rel="external" target="_blank">@messages("UTRSentByPost.link")</a> @messages("UTRSentByPost.paragraph2")</p>
+<p><a id="help-link" href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/trusts" target="_blank">@messages("UTRSentByPost.link.contactHelpline")</a> @messages("UTRSentByPost.paragraph2")</p>
+
```

Comment:
The link "Contact the trust helpline" opens a new tab or window, however the  text does not specify this. Please check with Anita / Rachel if this needs to be added.

Reply:
Anita and Rachel have confirmed the link needs the additional content.  (opens in a new window or tab) added to link and rendering on the page.